pandas/io/packers.py

```diff
@@ -595,11 +595,14 @@ def decode(obj):
         pd_dtype = pandas_dtype(dtype)
         np_dtype = pandas_dtype(dtype).base
 
+        # see gh-14901
+        klass_dtype = np_dtype if pd_dtype.name != 'category' else pd_dtype
+
         index = obj[u'index']
         result = globals()[obj[u'klass']](unconvert(obj[u'data'], dtype,
                                                     obj[u'compress']),
                                           index=index,
-                                          dtype=np_dtype,
+                                          dtype=klass_dtype,
```

Comment:
ahh ok, so & maybe should switch this to ``_values`` which avoids this problem (though maybe cause other issues)